src/app/organisations/data/organisationsStorage.js

```diff
@@ -149,7 +149,7 @@ const getOrgById = async (id) => {
   return org;
 };
 
-const pagedSearch = async (criteria, pageNumber = 1, pageSize = 25, filterCategories = undefined, filterStates = undefined) => {
+const pagedSearch = async (criteria, pageNumber = 1, pageSize = 25, filterCategories = undefined, filterStates = undefined, filterOutOrgNames = undefined) => {
```

Comment:
if filterOutOrgNames is defaulted to [] (see next comment)

Reply:
Done